target/linux/ramips/image/mt7621.mk

```diff
@@ -88,6 +88,13 @@ define Device/hc5962
 endef
 TARGET_DEVICES += hc5962
 
+define Device/k2p
+  DTS := K2P
+  IMAGE_SIZE := 16056320
```

Comment:
As far as I know, the variable isn't related to the Ralink SDK. It is the default firmware partition size for a lot of the support boards with 16MByte flash. Even for your board ralink_default_fw_size_16M matches. Please use `IMAGE_SIZE := $(ralink_default_fw_size_16M)` instead of the former requested 15744k.